bluebrain/repo-bluebrain/packages/spykfunc/package.py

```diff
@@ -15,10 +15,10 @@ class Spykfunc(PythonPackage):
 
     submodules = True
 
-    version("develop", branch="main")
+    version("develop", branch="main", deprecated=True)
```

Comment:
Just out of curiosity: what does `develop` deprecated mean here?

Reply:
It'll keep you from building it :)

I would've liked to deprecate everything, but then Spack will refuse to build our stack.